src/compiler/types/tests/generate-app-types.spec.ts

```diff
@@ -1406,6 +1409,198 @@ declare module "@stencil/core" {
                 }
         }
 }
+`,
+      {
+        immediateWrite: true,
+      }
+    );
+  });
+
+  it('should transform aliased paths if the config option is true', async () => {
```

Comment:
nit - can we add the name of the config option in the test here?
```suggestion
  it('should transform aliased paths if transformAliasedImportPaths is true', async () => {
```